src/KeyworkerManagement/index.js

```diff
@@ -43,6 +44,10 @@ class HomePage extends Component {
               <Link id="keyworker_settings_link" title="Key worker settings" className="link" to="/admin/settings" >Manage key worker settings</Link>
               <div className="padding-right-large">Allow auto-allocation. Edit key worker capacity and session frequency.</div>
             </div>}
+            {showMaintainRoles && <div className="pure-u-md-5-12">
+              <Link id="maintain_roles_link" title="maintain access roles" className="link" to="/maintainRoles/search" >Maintain access roles</Link>
```

Comment:
We'll need to protect the page at some point too. Just in case they get the URL

Reply:
yes - i was thinking about that